librz/asm/arch/rx/rx_datasheet.c

```diff
@@ -0,0 +1,306 @@
+// SPDX-FileCopyrightText: 2024 heersin <teablearcher@gmail.com>
```

Comment:
I am not sure `datasheet` is a good file name. Maybe `opcodes` is better?

Reply:
what about opcode_detail or opcode_desc ?

Reply:
Yes, that would be ok